src/bit-components.js

```diff
@@ -150,7 +150,12 @@ MediaPDF.map = new Map();
 export const MediaVideo = defineComponent({
   autoPlay: Types.ui8
 });
-export const AnimationMixer = defineComponent();
+export const MixerAnimatable = defineComponent({});
+// Map<EntityId, {mixer: AnimationMixer: animations: Array<AnimationClip>}>
```

Comment:
Not sure exactly the syntax but typescript can pickup on jsdoc comments for type info, so we might be able to use that here.  Similar to what we did here https://github.com/mozilla/hubs/blob/c78cec579eb422521e06cf1459469860277a47cd/src/components/gltf-model-plus.js#L818-L823